handler/area_types.go

```diff
@@ -187,6 +188,64 @@ func (h *AreaTypes) GetParents(w http.ResponseWriter, r *http.Request) {
 	h.respond.JSON(ctx, w, http.StatusOK, resp)
 }
 
+// GetParentAreaCount is the handler for /population-types/{population-type}/area-types/{area-type}/parents
+func (h *AreaTypes) GetParentAreaCount(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	areas := strings.Split(r.URL.Query().Get("areas"), ",")
+	req := contract.GetParentAreaCountRequest{
+		PopulationType: chi.URLParam(r, "population-type"),
+		AreaType:       chi.URLParam(r, "area-type"),
+		ParentAreaType: chi.URLParam(r, "parent-area-type"),
+		Areas:          areas,
+	}
+
+	if err := parseRequest(r, &req); err != nil {
+		h.respond.Error(ctx, w, http.StatusBadRequest, &Error{
+			err: errors.Wrap(err, "invalid request"),
+		})
+		return
+	}
+
+	cReq := cantabular.GetParentAreaCountRequest{
+		Dataset:  req.PopulationType,
```

Comment:
Actually could we unload the parameters straight into this object rather than going through the extra request object?

Reply:
Good point, not needed. I saw this pattern and I thought we had agreed on it.
I have removed as I agree with your comments.